src/main/java/io/debezium/connector/yugabytedb/consistent/Message.java

```diff
@@ -49,6 +49,18 @@ public int compareTo(Message o) {
         return 0;
     }
 
+    @Override
+    public boolean equals(Object obj) {
+        Message o = ((Message) obj);
+        return this.tablet.equals(o.tablet)
```

Comment:
Should tablet and commitTime be sufficient? Also is it common to make sure `this` and `obj` are not the same while comparing two objects in java?

Reply:
Two messages who belong to the same txn for same tablet will have the same tablet ID and commitTime and thus it will not be sufficient.

Not sure about the other question if it's a common practice.